docs/open-source/qiskit-sdk-version-strategy.mdx

```diff
@@ -4,3 +4,141 @@ description: TODO - filler text so that the SEO check doesn't complain!
 ---
 
 # Qiskit SDK version strategy
```

Comment:
We also want to link to https://github.com/Qiskit/qiskit/blob/main/DEPRECATION.md to explain how contributors "do deprecations". It could be something like

```markdown
## Contributors: how to mark deprecations

Refer to the [deprecation guide](https://github.com/Qiskit/qiskit/blob/main/DEPRECATION.md) in the Qiskit SDK repository for instructions on how to add deprecations to the source code.
```

Or maybe an Admonition?